recipes/g2aero/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "G2Aero" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com//NREL/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e5673caaf7af576050ea79659ee3b1e2d641e77f16918633e2a9bfaddce51294
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
```

Comment:
You mention 3.6 upstream, are you sure? https://github.com/NREL/G2Aero/blob/57777670af86b9555693ca404e3046e9ccd32f10/setup.py#L40

Reply:
Ok so I just tested with 3.6 and G2Aero does install and pass the pytest locally. I've changed the requirements accordingly. Good catch!